vars/deployHelpers.groovy

```diff
@@ -11,7 +11,11 @@ import groovy.transform.Field
 @Field deployJobs = [
     buildfactory: "/ops/deployBuildfactory",
     advisor: "/ops/deployAdvisor",
+    approval: "/ops/deployApproval",
+    catalog: "/ops/deployCatalog",
     platform: "/ops/deployPlatform",
+    sources: "/ops/deploySources",
+    topological-inventory: "/ops/deployTopologicalInventory",
```

Comment:
Yeah, looks like this is not valid groovy, but it looks like quoting the key name will work.

Reply:
Updated to add quotes.